Domain/sdLitica.Events/Abstractions/IEvent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace sdLitica.Events.Abstractions
+{
+    public interface IEvent
```

Comment:
For simple entities like events, do we really need this separation between interface and implementation? I'm from architecture perspective I understand from where this comes from, but are we going to have many events sharing the same interface?

Reply:
I think so. This is interface is good since it defined that all and every event must have Id, Datetime and Name. This properties help mainly to trace the event when it is in memory, or bus, or whether it was consumed or not. This helps not to miss the flow of the event. When we will have Logs, these data will help us also in debugging.